skopt/utils.py

```diff
@@ -292,3 +292,75 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+
+def dims_aslist(params_space):
```

Comment:
nitpick: spell out params and dims. I'd prefer that despite it being more to type